### DISCUSSION TITLE
Bots disappeared after using the SetClientCharacter command

### DISCUSSION THREAD

**Discussion by Vlad64bit** [FakeFishGames/Barotrauma#14678] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The bots disappeared after the command
When using the SetClientCharacter command and saving the game, the character from whom I switched to another disappears and so a lot can disappear at once, but I need this command to take perks on bots, in short, how to use bots without this command in mp or how to make sure that they do not disappear after the command. Translated by yandex 

### Reproduction steps

1. start multiplayer game
2. use SetClientCharacter on bot 
3. use SetClientCharacter on your main character
4. save game
5. start game
6. bot disappeared

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've now fixed this in our private repo (https://github.com/FakeFishGames/Barotrauma-development/commit/a7a58a9ec7971a80428503e25f301f54204175f5), fix coming up in a future update.

It's worth noting though that you don't need to take control of a bot to manage their talents, you can select the bot in the tab menu to set their talents.

**Comment by Rokvach** [comment]
Tested the fix and its working correctly. Fix coming in the next update. Closing discussion.
